playwright_tests/E2E/query-mgmt.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect  } from '@playwright/test';
+import { test } from '../E2E/fixtures/create-fixture';
+import config from '../config';
+import * as faker from "faker";
+
+let queryName = ''
+let queryDescription='';
+let caseId='' ;
+
+test.describe('FPL Test Case Data for QueryManagement  @QM', () => {
+
+  test.beforeAll(async () => {
+    console.log(' ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  Setup Data required across these tests  ~~~~~~~~~~~~~~~~~~~~~~');
+    // caseId hardcoded for now - will need to script it once availble on AAT.
+    // caseID  on https://xui-fpl-case-service-pr-5582.preview.platform.hmcts.net/cases
+    // Note: this is a long running pr branch of FPL and cases are not available once pods are refreshed.
+    caseId = '1730145870311253'; //
```

Comment:
The test should ideally create its own case. Otherwise we have to scramble round and recreate test cases when old data gets cleaned up